dlt/destinations/impl/clickhouse/factory.py

```diff
@@ -91,6 +91,23 @@ def from_destination_type(
 
         return super().from_destination_type(db_type, precision, scale)
 
+    def to_db_integer_type(self, column: TColumnSchema, table: Any = None) -> str:
```

Comment:
Have you looked into `PreparedTableSchema` instead of `Any` here?

Reply:
Thanks for catching this